warehouse/models/gtfs_views_staging/agency_clean.sql

```diff
@@ -22,7 +22,7 @@ agency_clean AS (
         TRIM(agency_email) AS agency_email,
         calitp_extracted_at,
         calitp_hash,
-        FARM_FINGERPRINT(CONCAT(CAST(calitp_hash AS STRING), "___", CAST(calitp_extracted_at AS STRING))) AS agency_key,
+        {{ farm_surrogate_key(['calitp_hash', 'calitp_extracted_at']) }} AS agency_key,
```

Comment:
can we just go ahead and use the macro in all the `gtfs_views_staging` tables? I think this pattern is the same everywhere.

Reply:
done!